files/database/postgresql/module.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
```

Comment:
I don't think this file is necessary, looks like your using the template and not this file.